sql/core/src/test/resources/sql-tests/inputs/natural-join.sql

```diff
@@ -1,8 +1,3 @@
--- List of configuration the test suite is run against:
```

Comment:
This is to test the analyzer/optimizer. Natural join will be rewritten to other normal joins.